src/pipedream/actions/types/__init__.py

```
@@ -0,0 +1,34 @@
+        module = import_module(module_name, __package__)
+        if module_name == f".{attr_name}":
+            return module
+        else:
+            return getattr(module, attr_name)
+    except ImportError as e:
+        raise ImportError(f"Failed to import {attr_name} from {module_name}: {e}") from e
+    except AttributeError as e:
+        raise AttributeError(f"Failed to get {attr_name} from {module_name}: {e}") from e
+
+
+def __dir__():
```

Ruff (0.14.8)

Missing return type annotation for private function `__dir__`

(ANN202)